### ISSUE TITLE
blt doesn't work under certain circumstances

### ISSUE THREAD

**Issue by adamboyers** [kitao/pyxel#62]
The following code fails to `blt` properly.  Instead of `blt`-ing the bitmap `set` in `Game.__init__`, it `blt`'s all black pixels.

Oddly enough, the following things all correct this error:
- Setting `App.mode` to `Game()` instead of `Menu()` from the beginning
- Replacing body of `Menu.draw` with `pass`
- Adding any reference to `pyxel.image(0).data` to `Game.init`

```python
import pyxel

class Game:
    def __init__(self):
        pyxel.image(0).set(
            0, 0,
            [
                '0123',
                '4567',
                '89ab',
                'cdef'
            ]
        )

    def update(self):
        pass

    def draw(self):
        pyxel.cls(12)

        pyxel.blt(
            0, 0,
            0,
            0, 0,
            4, 4,
        )

class Menu:
    def __init__(self, app):
        self.app = app

    def update(self):
        if pyxel.btnp(pyxel.KEY_ENTER):
            self.app.mode = Game()

    def draw(self):
        pyxel.cls(5)
        pass

class App:
    def __init__(self):
        pyxel.init(16, 16)

        self.mode = Menu(self)

        pyxel.run(self.update, self.draw)

    def update(self):
        self.mode.update()

    def draw(self):
        self.mode.draw()

App()
```

**Comment by kitao** [comment]
Sorry for late reply.
I started checking what is happening by running your code.

**Comment by kitao** [comment]
I found the reason.
I forget to change the update flag of texture when the set method is called.
I'll fix this in the Pyxel 0.8.2.

**Comment by kitao** [comment]
I fixed it in 0.8.2.
